miniapps/electromagnetics/makefile

```diff
@@ -76,6 +76,10 @@ RUN_MPI = $(MFEM_MPIEXEC) $(MFEM_MPIEXEC_NP) $(MFEM_MPI_NP)
 volta-test-par: volta
 	@$(call mfem-test,$<, $(RUN_MPI), Electromagnetic miniapp,\
 	-maxit 2 -dbcs 1 -dbcg -ds '0.0 0.0 0.0 0.2 8.0')
+volta-1-test-par: volta
+	@$(call mfem-test,$<, $(RUN_MPI), Electromagnetic miniapp,\
+	-maxit 2 -m ../../data/square-disc.mesh \
+	-dbcs '1 2 3 4 5 6 7 8' -dbcv '0 0 0 0 1 1 1 1')
```

Comment:
As it is, the target `volta-1-test-par` is not run by `make test`. To run multiple tests of the same app, you can use something like this:
```makefile
volta-test-par: volta-test-1 volta-test-2
volta-test-1: volta
	@$(call mfem-test,$<, $(RUN_MPI), Electromagnetic miniapp,\
	-maxit 2 -dbcs 1 -dbcg -ds '0.0 0.0 0.0 0.2 8.0')
volta-test-2: volta
	@$(call mfem-test,$<, $(RUN_MPI), Electromagnetic miniapp,\
	-maxit 2 -m ../../data/square-disc.mesh \
	-dbcs '1 2 3 4 5 6 7 8' -dbcv '0 0 0 0 1 1 1 1')
```

Reply:
Thanks, @v-dobrev , that makes sense.

Reply:
@mlstowell, if you want you can add this in https://github.com/mfem/mfem/pull/1314

Reply:
Thanks, it has been done.